src/main/java/baza/trainee/controller/EventAdminController.java

```diff
@@ -0,0 +1,75 @@
+package baza.trainee.controller;
+
+import baza.trainee.domain.dto.event.EventDto;
+import baza.trainee.domain.dto.event.EventPreviewDto;
+import baza.trainee.domain.dto.event.EventPublicationDto;
+import baza.trainee.service.EventService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admin/events")
+@RequiredArgsConstructor
+public class EventAdminController {
+
+    private final EventService eventService;
+
+    /**
+     * @return  A list of event previews.
+     */
+    @GetMapping
+    public ResponseEntity<List<EventPreviewDto>> getEvents() {
+        List<EventPreviewDto> events = eventService.getEvents();
+        return ResponseEntity.ok(events);
+    }
```

Comment:
it can be simplified to
```suggestion
    public List<EventPreviewDto> getEvents() {
        return eventService.getEvents();
    }
```

Reply:
fixed: now get events from service and convert using converter to dto